mfhdf/hdiff/hdiff.h

```diff
@@ -4,13 +4,10 @@
  *   /hdf/src/master/mfhdf/ncdump/ncdump.h,v 1.1 1993/04/21 21:51:19 chouck Exp
  *********************************************************************/
 
-
-
-
 #ifndef HDIFF_H__
 #define HDIFF_H__
 
-
+#include <unistd.h>
```

Comment:
I feel like this should go in the main header file. This will also fail on Windows, so it needs to be ifdef-protected.